webrender/Cargo.toml

```diff
@@ -22,6 +22,7 @@ fnv = "1.0"
 gleam = "0.4.7"
 lazy_static = "0.2"
 log = "0.3"
+nanovg = {version = "0.3", features = ["gl2"]}
```

Comment:
yep, and it needs to decide whether use OpenGL or OpenGL ES during compile time because only one of them can be enable at a time. Will try to fix.